### DISCUSSION TITLE
Log file is empty on Debian 11

### DISCUSSION THREAD

**Discussion by adevoss** [inspircd/inspircd#1993] [Support]
After installing on Debian 11 inspircd is running.
In the configuration file I put:

`<log method="file" type="*" level="default" target="/var/log/inspircd/inspircd.log">`
but the file remains empty.

The file:
`-rw-r--r-- 1 irc irc 0 Jul 20 16:26 /var/log/inspircd/inspircd.log`

Change ownership from root to irc didn't help.
How do I configure a log file?

**Comment by sadiepowell** [comment]
I created a fresh install in a docker container using our latest package and that log tag and it works for me.

Would it be possible for you to tell us how you installed InspIRCd, what version you're using, and how you started it?

**Reply by adevoss** [reply]
In a VM (amd64) on proxmox:
`apt install inspircd`
Version: 3.8.1-2

Started server by executing:
`service inspircd (re)start`
Modified the config file. IRC server runs and modifications like name, description and bind address take effect.
```
ps -ef|grep inspircd
irc         3809       1  0 Jul20 ?        00:00:02 /usr/sbin/inspircd --nofork --nopid
```

**Reply by sadiepowell** [reply]
That version is packaged by Debian so you'll need to file an issue about it here as we don't support third-party packages: https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=inspircd;dist=stable

However, you might find it easier to use our official Debian packages though as they're more reliable. Here's the one for your system: https://github.com/inspircd/inspircd/releases/download/v3.13.0/inspircd_3.13.0.deb11u1_amd64.deb

**Reply by adevoss** [reply]
Thanks for pointing me to the latest stable release.

After installing some new dependencies and upgrading to this release there was no change.
A reboot did help. I was surprised this was needed on Linux.

So I don't know if it was the new version or the reboot that solved this.
Thanks for helping.
